branchpro/apps/_dash_app.py

```diff
@@ -139,14 +139,12 @@ def _read_uploaded_file(self, contents, filename, is_si=False):
                     data = pd.read_csv(
                         io.StringIO(decoded.decode('utf-8')),
                         header=None)
-                    if len(data.columns) > 1:
-                        return html.Div(['Incorrect format; file must be \
-                            1-dimensional']), None
-                    else:
-                        data = data.values[:, 0]
-                    if isinstance(data[0], str) and not data[0].isnumeric():
-                        return html.Div(['Incorrect format; file must not have a \
-                        header']), None
+                    data = data.fillna(0).values
+                    for _ in range(data.shape[1]):
+                        if isinstance(data[0, _], str) and \
+                                not data[0, _].isnumeric():
+                            return html.Div(['Incorrect format; file must not have a \
+                            header']), None
```

Comment:
If the user uploads multiple serial intervals, it would be helpful for the message to say "Loaded data (N samples) from:..."---this will inform them that their file was read correctly and remind them that the slower type of inference is running.